src/rpc/rawtransaction.cpp

```diff
@@ -322,10 +322,10 @@ static RPCHelpMan getrawtransaction()
     const NodeContext& node = EnsureAnyNodeContext(request.context);
     ChainstateManager& chainman = EnsureChainman(node);
 
-    uint256 hash = ParseHashV(request.params[0], "parameter 1");
+    auto txid{Txid::FromUint256(ParseHashV(request.params[0], "parameter 1"))};
```

Comment:
nit, if you have to retouch: could rename this one to "txid" as well (and with s/parameter 3/blockhash/ below, we got them all)